OMEdit/OMEditGUI/TransformationalDebugger/OMDumpXML.cpp

```diff
@@ -249,24 +249,26 @@ QString OMEquation::toString()
 {
   if (tag == "dummy") {
     return "";
-  } else if (tag == "assign") {
+  } else if (tag == "assign" || tag == "torn" || tag == "jacobian") {
     if (text.size()==1) {
-     return QString("%1 := %2").arg(defines[0]).arg(text[0]);
+     return QString("(%1) %2 := %3").arg(tag).arg(defines[0]).arg(text[0]);
     } else {
-     return QString("%1 := %2").arg(text[0]).arg(text[1]);
+     return QString("(%1) %2 := %3").arg(tag).arg(text[0]).arg(text[1]);
     }
   } else if (tag == "statement" || tag == "algorithm") {
     return text.join("\n");
-  } else if (tag == "container") {
+  } else if (tag == "system") {
     return QString("%1, size %2").arg(display).arg(eqs.size());
```

Comment:
Please do the same for `system` as you did for `tornsystem` for consistency:
`nonlinear, unknowns: 5, iteration variables: 5`
`eqs.size()` is not the actual system size.